src/App.jsx

```diff
@@ -9,32 +10,20 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
+export const App = () => {
+  const [value, setValue] = useState({
+    id: 'tab-1', title: 'Tab 1', content: 'Some text 1',
+  });
```

Comment:
Rename value to be more descriptive.
![image](https://github.com/mate-academy/react_tabs-js/assets/99403198/bfc217f9-cf2c-4ab6-ae4f-8d517ff70ad3)
```suggestion
  const [value, setValue] = useState(tabs[0].id);
```